### PR TITLE
Removing explicit use of term "SLA"

### PR DESCRIPTION
**Pull request by anwara-ms**
We dont have any SLA for Retention

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/retention-policies.md [hunk 1]
```diff
@@ -188,7 +188,7 @@ The following are known issues for retention policies in Teams that are being tr
 
 - Under **Choose users** in the **Teams chats** location row, you may see guests and non-mailbox users. Retention policies aren't meant to be set for guests, and we're working to remove these from the list.
 
-- Exchange Life Cycle assistant (ELC) runs daily, but it has an SLA of 7 days. As a result, it's possible that, if you have a Teams retention policy to delete items older than 60 days, these items could persist for up to 67 days. This isn't a new situation - it follows the Exchange model. Of course, in most cases, there's no delay.
+- Exchange Life Cycle assistant (ELC) runs daily, but the latency is known to have run upto 7 days, in some cases. As a result, it's possible that, if you have a Teams retention policy to delete items older than 60 days, these items could persist for up to 67 days. This isn't a new situation - it follows the Exchange model. Of course, in most cases, there's no delay.
```
Comment (JohanFreelancer9) [inline Teams/retention-policies.md:191]:
```suggestion
- Exchange Life Cycle assistant (ELC) runs daily, but the latency is known to have run for up to 7 days in some cases. As a result, it's possible that, if you have a Teams retention policy to delete items older than 60 days, these items could persist for up to 67 days. This isn't a new situation - it follows the Exchange model. Of course, in most cases, there's no delay.
```

**Comment by scanum** [comment]
@cabailey PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by cabailey** [comment]
Accepting this PR so we can remove the SLA wording but I question the helpfulness of the wording in the entire bullet - sudden introduction of Exchange Life Cycle assistant is unclear and looks like applicable only if items are older than 60 days.  Let's fix this with a later version.

**Pull request merged**